### PR TITLE
feat: [SEI-3702]: Harness App logs out while selecting project in SEI module

### PR DESCRIPTION
**Pull request by sharath-levelops**
### Summary

organization API was throwing error 401, since the account id is not able retrieve from path params. So, added fallback account id as props

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### CODE REVIEW

**Comment by sharath-levelops** [comment]
trigger messagecheck

**Comment by sharath-levelops** [comment]
retrigger titlecheck

**Comment by sharath-levelops** [comment]
trigger messagecheck

**Comment by sharath-levelops** [comment]
retrigger titlecheck

**Comment by sharath-levelops** [comment]
trigger messagecheck

File: src/modules/10-common/OrgDropdown/__tests__/OrgDropdown.test.tsx [hunk 1]
```diff
@@ -103,4 +103,16 @@ describe('org dropdown test', () => {
...
+  test('test when account id is not available in path props ', () => {
+    orgListPromiseMock = jest.fn().mockImplementation(() => {
+      return Promise.resolve(undefined)
+    })
+    const { queryByText } = render(
+      <TestWrapper pathParams={{ accountId: '' }}>
+        <OrgDropdown onChange={noop} fallbackAccountId="dummy" />
+      </TestWrapper>
+    )
+    expect(queryByText('orgDropdown')).not.toBeNull()
```
Comment (abhinavrastogi-harness) [inline src/modules/10-common/OrgDropdown/__tests__/OrgDropdown.test.tsx:116]: this test is not actually verifying the behaviour, right?
`queryByText('orgDropdown')` will always be defined. 
what we need to test is if the api call gets made with the fallback account id.

**Review by abhinavrastogi-harness — APPROVED** [review]

**Review by sahithikolichala — APPROVED** [review]

**Pull request merged**